Gruntfile.js

```diff
@@ -215,6 +215,9 @@ const configureGrunt = function (grunt) {
                     var upstream = grunt.option('upstream') || process.env.GHOST_UPSTREAM || 'upstream';
                     grunt.log.writeln('Pulling down the latest master from ' + upstream);
                     return `
+                        git submodule sync
+                        git submodule update
```

Comment:
This makes sure that if it's just the submodule being out of date then it doesn't get caught in the `git diff` check